Sources/Swift/Integrations/Performance/SentryProfileOptions.swift

```diff
@@ -42,8 +46,10 @@ public class SentryProfileOptions: NSObject {
     
     /// The % of user sessions in which to enable profiling.
     /// - warning: Continuous profiling is an experimental feature and may still contain bugs.
-    /// - note: Whether or not the session is sampled is determined once, when the SDK is initially
-    /// configured.
+    /// - note: The decision whether or not to sample profiles is computed using this sample rate
+    /// when the SDK is started, and applies until the app resigns its active status. It is then
+    /// reevaluated on subsequent launch events that start the SDK. The time between these two
```

Comment:
> It is then reevaluated on subsequent launch events that start the SDK
    
It's not clear to me from this that it would be re-evaluated upon foregrounding, since that doesn't "start the SDK" (which only happens on launch)